### PR TITLE
{bio}[GCCcore/10.3.0] multichoose v1.0.3

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
node163.pri.viking.alces.network - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/8e98917c3b267758cf038b2010c2350f for a full test report.

File: easybuild/easyconfigs/m/multichoose/multichoose-1.0.3-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+builddependencies = [('binutils', '2.36.1')]
+
+local_bins = [name, 'multipermute']
+files_to_copy = [(local_bins, 'bin'), (['*.h'], 'include/multichoose')]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in local_bins],
+    'dirs': [],
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/m/multichoose/multichoose-1.0.3-GCCcore-10.3.0.eb:25]: @verdurin Though `multichoose --help` doesn't really seem to work (it returns exit code 1), would it make sense to use the example from their README.md as sanity check command?
```
multichoose 2 a t g c
a a 
a t 
a g 
a c 
t t
t g 
t c 
g g 
g c 
c c
```
Or even shorter, `multichoose 1 a`?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.7
See https://gist.github.com/8c4c4f96a8eb4a2aba5e259b88b1c99d for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.7
See https://gist.github.com/02f321b153cbda5349d4cb3ed9fd96e5 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/caadc4dc486ee69b4d552ee1d30632c1 for a full test report.

File: easybuild/easyconfigs/m/multichoose/multichoose-1.0.3-GCCcore-10.3.0.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/m/multichoose/multichoose-1.0.3-GCCcore-10.3.0.eb:25]: @bedroge agreed - now added.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/439d7681557aefadada5fd0369942dac for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/8d32fc418bbf5d1228efe2d54a064f5c for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @verdurin!

**Pull request merged**